portfolio/src/main/java/com/google/sps/servlets/DeleteCommentsServlet.java

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** 
+ * Servlet that deletes comments from Datastore.
+ */
+@WebServlet("/delete-comments")
+public class DeleteCommentsServlet extends HttpServlet {
+
+  /**
+   * On POST request, deletes all comments currently stored in the server's datastore.
+   * @param request The request made by the connecting client.
+   * @param response The response that is sent back to the client.
```

Comment:
I recommend checking out @inheritDoc, which should make these easier to maintain and write.
https://docs.oracle.com/javase/6/docs/technotes/tools/solaris/javadoc.html#@inheritDoc
I recommend reaching out to zghera about them, as he has been utilizing them well recently.

Reply:
Oh I didn't know about this for javadoc, I'll ask zghera about them more, thanks!

Reply:
Okay so I didn't know Javadoc will inherit documentation from the superclass of an overridden method if none is provided, so I just went and removed the @params.